src/church.html

```diff
@@ -32,7 +32,10 @@ <h1 id="church-title"></h1>
 					<a id="church-wikipedia-link" href="#">Läs mer på Wikipedia</a>
 					<h2 id="bbr-heading">Bebyggelseregistret</h2>
 					<p id="church-bbr" class="expandable-p"></p>
+
 					<button id="bbr-expand">Expandera</button>
+					<a id="church-report" class="hidden" href="#">Anmärk<span class="entypo-edit"></span></a>
```

Comment:
@rjakobsson could we go without a icon for now, I will probably introduce Font Awesome later on in a lot of places...